src/domain.cpp

```diff
@@ -984,22 +984,34 @@ void Domain::subbox_too_small_check(double thresh)
      use minimum_image_once() instead
 ------------------------------------------------------------------------- */
 
+#ifdef LAMMPS_BIGBIG
+static constexpr double MAXIMGCOUNT = 1<<21;
```

Comment:
Why not make MAXIMGCOUNT much smaller, e.g. 1024 or even 4.
Are there cases when a value > 4 between a pair of atoms is not
an indication of a bad dynamics ?

Reply:
Fine with me.

I made a conservative choice by looking at the maximum possible distance due to image flags. Anything beyond that is not representable in LAMMPS.

> Are there cases when a value > 4 between a pair of atoms is not an indication of a bad dynamics ?

I was trying to come up with the worst possible case, e.g. people that put together systems from individually equilibrated data files and then try to cross-link monomers into polymers.

Reply:
let's use 16
the comment for the method indicates it can be needed when the cutoff > boxsize,
  16 seems a reasonable limit on that

Reply:
also in the comment header for this method, on the last line can you
change "use" to "it uses" and indent it another 2 spaces
it is referring to fix_shake using minimum_image_once()

Reply:
@sjplimp updates are made. Please also note the new "reset_image_flags" command, which should help folks to reset their systems so that the image flags are zero or close it it. A good test is the "peptide" example input which has water molecules that have passed through the box multiple times. This resets them so that at least one atom of a water has all zero image flags.

Reply:
The command is nearly a macro, since most of the work is done by several existing computes and variables.

Reply:
Useful functionality but I'm not keen on adding new commands if we can avoid it.  Doesn't have to be in this PR,
but what if we combined reset_atom_ids, reset_mol_ids, reset_image_flags into a single reset command.  Similar to
"set" it (re)sets attributes of atoms.  Reset_timestep could stay, b/c it is widely used and is not resetting atom properties.

Reply:
OK. I will take the commits out of this PR and create a new one. While we are looking at combining commands, could we perhaps add the "box" command functionality somewhere, too? It has only one option and that is a bit obscure. I only had to use it once recently when `replicate` on a triclinic box mysteriously triggered a "too large tilt" error.